src/community/blocks.tsx

```diff
@@ -0,0 +1,267 @@
+import * as React from 'react';
+import Stack from '../stack';
+import * as styles from './blocks.css';
+import {Text2, Text3, Text5, Text8} from '../text';
+import {vars} from '../skins/skin-contract.css';
+import Inline from '../inline';
+import Box from '../box';
+import ProgressBar from '../progress-bar';
+import classNames from 'classnames';
+
+import type StackingGroup from '../stacking-group';
+import type Image from '../image';
+import type Tag from '../tag';
+import type {RendersNullableElement} from '../utils/renders-element';
+import type {ExclusifyUnion} from '../utils/utility-types';
+
+interface BlockContentProps {
+    title?: string;
+    description?: Array<string>;
```

Comment:
Please @BrunoGuimaraes103 , I've commented this multiple times, this description shouldn't be an `Array`, it should be just a `string`

Reply:
fixed